src/components/ApplicationGrid/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import Paper from '@material-ui/core/Paper';
+import {
+  SearchState,
+  IntegratedFiltering,
+} from '@devexpress/dx-react-grid';
+import {
+  Grid,
+  Table,
+  TableColumnVisibility,
+  Toolbar,
+  ColumnChooser,
+  SearchPanel,
+  TableHeaderRow,
+} from '@devexpress/dx-react-grid-material-ui';
+import { getAllApplications } from "../../utils/API/index.js";
+
+const getHiddenColumnsFilteringExtensions = hiddenColumnNames => hiddenColumnNames
+  .map(columnName => ({
+    columnName,
+    predicate: () => false,
+  }));
+
+export default  () => {
+  
+    const [rows, setRows] = useState([]);
+    getAllApplications(
```

Comment:
If you're using vscode you can install the extension [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) and set it up so that when you hit save it automatically formats your file all nicely!

Reply:
Ran it thru Prettier and updated the formatting of the files changed with a new commit